packages/vue-email/src/components/e-button.vue

```diff
@@ -1,72 +1,52 @@
 <template>
-  <a
-    v-bind="$props"
-    data-id="__vue-email-button"
-    :href="href"
-    :target="target"
```

Comment:
Usually the email buttons redirect to a blank page loading the target, both attributes must be required, what do you think about it?